specification/network/resource-manager/Microsoft.Network/stable/2024-05-01/virtualNetworkGateway.json

```diff
@@ -1231,6 +1231,288 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworkGateways/{virtualNetworkGatewayName}/getFailoverAllTestsDetails": {
+      "post": {
+        "tags": [
+          "VirtualNetworkGateways"
+        ],
+        "operationId": "VirtualNetworkGateways_GetFailoverAllTestDetails",
+        "description": "This operation retrieves the details of all the failover tests performed on the gateway for different peering locations",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "virtualNetworkGatewayName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the virtual network gateway."
+          },
+          {
+            "name": "type",
```

Comment:
>type [](http://example.com/codeflow?start=21&length=4)

Have you considering using a better (explicit) name?

Reply:
This change is already in NRP, so can't change now